frame/referenda/src/types.rs

```diff
@@ -259,6 +261,9 @@ pub enum Curve {
 	SteppedDecreasing { begin: Perbill, end: Perbill, step: Perbill, period: Perbill },
 	/// A recipocal (`K/(x+S)-T`) curve: `factor` is `K` and `x_offset` is `S`, `y_offset` is `T`.
 	Reciprocal { factor: FixedI64, x_offset: FixedI64, y_offset: FixedI64 },
+	/// Exponential decay curve defined by `f(x) = n_0 * (2 ^ -(x / half_time)) + offset`
+	/// https://en.wikipedia.org/wiki/Exponential_decay
+	ExponentialDecay { n_0: I64F64, half_time: I64F64, offset: I64F64 },
```

Comment:
probably we want `half_time` to either be 

* of type `T::BlockNumber`
* a fraction of the entire voting period (preferred)

please add very clear comments about the units and meanings of each parameter

Reply:
not perbill, no. just add comments explaining exactly this mapping to "voting period = 1"

Reply:
actually, I think we want them to adopt substrate-fixed for many other cases whare currently perbill is used. But one step at the time. We should just make the example with substrate-fixed